### ISSUE TITLE
automated generation of dmb citation hint

### ISSUE THREAD

**Issue by AndyDaniel1** [dzhw/metadatamanagement#2786]
Since all the required information for the citation guidelines is stored in the mdm, we should implement an automated generation of the citation guidelines (using Word or LaTeX templates).

**Comment by UteH** [comment]
I fully support this idea for the data packages🎉 

note: Currently, we also list the citation of the methods report. We do _not_ have this information in the MDM. Thus, we would have do add/list this citation manually(?)

**Comment by rreitmann** [comment]
We think we can improve the metadata entered for the dataPackage-Attachment "Daten- und Methodenbericht" in order to offer an automatically generated citation hint for public users.

**Comment by svietgen** [comment]
I tested this issue with repo "gra2005" and everything worked!
Subsequent changes were displayed correctly and when I added a second report in a different language (english), both were displayed regarding the language I chose in the MDM. 

As long as we don't 
- have more than one report with the same language (because only one is displayed)
- have a report in a language other than german or english      

everything works.

Currenty the button says "Cite..."/"Zitieren...". Can we change this?
Suggestion (but maybe a bit too long):

![cite](https://user-images.githubusercontent.com/22055122/115719399-56a72980-a37c-11eb-9be5-4a0b00a6b64f.png)
